src/js/repeat.js

```diff
@@ -199,9 +199,10 @@ export default {
     updateDefaultFirstRepeatInstance( repeatInfo ) {
         const repeatPath = repeatInfo.dataset.name;
         if ( !this.form.model.data.instanceStr && !this.templates[ repeatPath ].classList.contains( 'or-appearance-minimal' ) ) {
-            const repeatSeriesIndex = this.getIndex( repeatInfo );
-            const repeatSeriesInModel = this.form.model.getRepeatSeries( repeatPath, repeatSeriesIndex );
-            if ( repeatSeriesInModel.length === 0 ) {
+            const prevSibling = repeatInfo.previousElementSibling;
+            let repeatIndexInSeries = prevSibling && prevSibling.classList.contains( 'or-repeat' ) ?
+                Number( prevSibling.querySelector( '.repeat-number' ).textContent ) : 0;
+            if ( repeatIndexInSeries === 0 ) {
```

Comment:
I'm a little lost with what is happening here. Could you explain please? Did you consider fixing this in model.getRepeatSeries instead? Doesn't that function have a bug that you discovered here? If so, would it be best to fix that function instead of working around it, or am I misunderstanding?

Reply:
seems these changes were to get around the error in the `getIndex` function. not needed anymore